ios/Podfile.lock

```diff
@@ -3932,4 +3938,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 529190a386115c6ed272c0477f2ee8e0ce0b7490
 
-COCOAPODS: 1.8.4
+COCOAPODS: 1.9.1
```

Comment:
Would you mind running `bundle install` in the root of the repository? It should install CocoaPods in the correct version.

Reply:
running `bundle install` resulted in v 1.9.1 still being used 🤔